resources/definitions/fokoos_odinsmart.def.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name":      "Fokoos Odinsmart",
+    "version":   2,
+    "inherits":  "fokoos_base",
+    "metadata":  {
+                  "author": "fokoos.com",
```

Comment:
Please fix the formatting (again, this will make it easier to maintain in the future)

Reply:
have done formatting